src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -3919,5 +3919,1172 @@ class Test1 : I1
+@"
+public interface I1
+{
+    event System.Action E1 
+    {
+        get;
+        set;
+    } => 0;
+}
+
+class Test1 : I1
+{}
+";
+            var compilation1 = CreateCompilationWithMscorlib(source1, options: TestOptions.DebugDll,
+                                                             parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.Latest));
+            Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+
+            compilation1.VerifyDiagnostics(
+                // (8,7): error CS1519: Invalid token '=>' in class, struct, or interface member declaration
+                //     } => 0;
+                Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "=>").WithArguments("=>").WithLocation(8, 7),
+                // (6,9): error CS1055: An add or remove accessor expected
+                //         get;
+                Diagnostic(ErrorCode.ERR_AddOrRemoveExpected, "get").WithLocation(6, 9),
+                // (7,9): error CS1055: An add or remove accessor expected
+                //         set;
+                Diagnostic(ErrorCode.ERR_AddOrRemoveExpected, "set").WithLocation(7, 9),
+                // (4,25): error CS0065: 'I1.E1': event property must have both add and remove accessors
+                //     event System.Action E1 
+                Diagnostic(ErrorCode.ERR_EventNeedsBothAccessors, "E1").WithArguments("I1.E1")
+                );
+
+            ValidateEventImplementationTest1_101(compilation1.SourceModule, haveAdd: false, haveRemove: false);
+
+            var source2 =
+@"
+class Test2 : I1
+{}
+";
+
+            var compilation2 = CreateCompilationWithMscorlib(source2, new[] { compilation1.ToMetadataReference() }, options: TestOptions.DebugDll);
+            Assert.True(compilation2.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+
+            void Validate2(ModuleSymbol m)
+            {
+                ValidateEventImplementationTest2_101(m, haveAdd: false, haveRemove: false);
+            }
+
+            Validate2(compilation2.SourceModule);
+
+            compilation2.VerifyDiagnostics();
+            CompileAndVerify(compilation2, verify: false, symbolValidator: Validate2);
+        }
+
+        [Fact]
+        public void EventImplementation_109()
+        {
+            var source1 =
+@"
+public interface I1
+{
+    event System.Action E1 
+    {
+        add => throw null;
+        remove;
+    }
+}
+
+class Test1 : I1
+{}
+";
+            var compilation1 = CreateCompilationWithMscorlib(source1, options: TestOptions.DebugDll,
+                                                             parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.Latest));
+            Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+
+            compilation1.VerifyDiagnostics(
+                // (6,12): error CS0073: An add or remove accessor must have a body
+                //         add;
```

Comment:
Feels fine to me and is not caused by the change. Abstract events must omit accessors list. Non-abstract custom events must provide bodies for both accessors. This behavior feel reasonable to me.